src/main/javascript/README.md

```diff
@@ -0,0 +1,20 @@
+Registration:
```

Comment:
Could we use the same readme as we have for java client + specific JS info?

Reply:
yeah, but readme for js need litle changes, in js we dont have these enums BikeType, GameElementType, SpringboardElementType; we have only Elements var